MARC.js

```diff
@@ -387,13 +387,14 @@ record.prototype.translate = function(item) {
 		item.title = title.a;
 		if(title.e) {
 			//If the title proper did not end in a punctuation mark, we should add a colon
-			if(item.title.search(/[A-Za-z0-9]\s*/) != -1) {
+			if (item.title.search(/[\?:,\.!;]\s*$/) == -1) {
```

Comment:
this is my bad from the other thread, but since @aurimasv is going to come and complain anyway:
There's no need to escape the question mark and the period:
> the only special characters or metacharacters inside a character class are the closing bracket (]), the backslash (\\), the caret (^), and the hyphen (-).